README.md

```
@@ -1,29 +1,19 @@
-# Indexer
+# Indexer + API app
+The project has 3 parts
+- Postgres DB - schema defined in `supabase` hosted by supabase, uses some of their tools
+- Indexer "ETL" go code in `go-indexer` - long-running server that
+    - connects to an ethereum node
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)